src/components/Books/AddBook.js

```diff
@@ -69,38 +70,47 @@ const AddNewBook = () => {
   }, [isTitleValid, isAuthorValid, title, author]);
 
   return (
-    <Card>
-      <h3>Add New Book</h3>
-      <form onSubmit={addBookHandler}>
-        <div className={!isTitleValid ? 'invalid' : ''}>
-          <p>{!isTitleValid && 'Book title should be 3 to 55 char long!'}</p>
+    <Card extraclass={classes.form_container}>
+      <h2>Add New Book</h2>
+      <form onSubmit={addBookHandler} className={classes.form}>
+        <div
+          className={`${classes.form_control} ${
+            !isTitleValid ? classes.input_invalid : ''
```

Comment:
- [ ] Please make sure that this line matches the design in color and width. 
your design 
![image](https://user-images.githubusercontent.com/74744787/222905391-487168bc-7823-4f79-8aac-f5697faf2029.png)
 Zeplin 
![image](https://user-images.githubusercontent.com/74744787/222905464-c0a6dcaa-2a44-4819-b9a5-c2f97082466e.png)